src/components/Pagination/index.jsx

```diff
@@ -7,21 +7,39 @@ const Pagination = ({
   nextPageIndex,
   updatePageIndex,
   pageIndex,
+  pageInterval,
   numRules,
 }) => {
   const endPage = pageIndex + (5 - (pageIndex % 5 === 0 ? 5 : pageIndex % 5));
   const startPage = endPage - 4;
   return (
     <div className="pagination pagination-container">
       <a
-        className="btn btn-secondary btn-small pagination-prev"
+        className={
+          `btn btn-secondary btn-small pagination-prev ${ 
+          pageIndex <= 5 ? "is-disabled" : ""}`
```

Comment:
instead of all these `5`s everywhere we can store that in a variable in case we ever need to change the number of pages to show at a time we would only need to change it in one place

Reply:
+1 ^